packages/expo-maps/ios/MapPermissionRequester.swift

```diff
@@ -0,0 +1,74 @@
+import ExpoModulesCore
+
+class MapPermissionRequester: NSObject, EXPermissionsRequester, CLLocationManagerDelegate {
+  private var locationManager = CLLocationManager()
+  private var _resolve: EXPromiseResolveBlock?
+  private var _reject: EXPromiseRejectBlock?
```

Comment:
```suggestion
  private var resolve: EXPromiseResolveBlock?
  private var reject: EXPromiseRejectBlock?
```
We try to avoid this convention. Just explicitly reference `self` where necessary